queries/html.scm

```diff
@@ -0,0 +1,93 @@
+;;!! <aaa>
+;;!   ^^^
+;;!  -----
+(start_tag
+    (tag_name) @name
+) @_.domain
+
+;;!! </aaa>
+;;!    ^^^
+;;!  ------
+(end_tag
+    (tag_name) @name
+) @_.domain
+
+;;!! <aaa id="me">
+;;!       ^^^^^^^
+(attribute) @attribute
+
+;;!! <aaa id="me">
+;;!       ^^
+(attribute
+    (attribute_name) @collectionKey
+) @_.domain
+
+;;!! <aaa value=2>
+;;!             ^
+(attribute
+    (attribute_name) @value.removal.start.endOf
+    (attribute_value) @value @value.removal.end.endOf
```

Comment:
For consistency we should prob switch all the removals to leading / trailing where possible

Reply:
Done